superset/config.py

```diff
@@ -1583,7 +1583,7 @@ class ExtraRelatedQueryFilters(TypedDict, total=False):
     try:
         # pylint: disable=import-error,wildcard-import,unused-wildcard-import
         import superset_config
-        from superset_config import *  # type: ignore
+        from superset_config import *
```

Comment:
without this, the mypy pre-commit hook blocks the commit. unsure this is the appropriate approach